samples/emoji-search/ios-uikit/EmojiSearchApp.xcodeproj/xcshareddata/xcschemes/EmojiSearchApp.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding a scheme here to enable us to Profile the EmojiSample app in Instruments

Reply:
Nice, thanks